akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -230,6 +250,29 @@ private[http2] class Http2ServerDemux(http2Settings: Http2CommonSettings, initia
         }
       })
 
+      /**
+       * Tune this peer to enforce the settings configured from this peer.
+       * @return TODO: it's a Boolean but I don't think it has to be.
+       */
+      private def enforceSettings(settings: immutable.Seq[Setting]): Boolean = {
+        var settingsAppliedOk = true
+
+        settings.foreach {
+          case Setting(Http2Protocol.SettingIdentifier.SETTINGS_MAX_CONCURRENT_STREAMS, value) =>
+          // nothing to do here: the count of open incoming streams
+          // depends on `def maxConcurrentStreams` above which is currently not modifiable
+        }
+
+        settingsAppliedOk
+      }
+
+      /**
+       * Tune this peer to honour the settings sent from the other peer.
```

Comment:
Added to disambiguate what settings this was referring to (the local settings VS the settings received from the peer).